data/index.js

```diff
@@ -139,7 +155,18 @@ export function select( reducerKey ) {
 		return select( reducerKey )[ selectorKey ]( ...args );
 	}
 
-	return selectors[ reducerKey ];
+	const createResolver = ( selector, key ) => ( ...args ) => {
+		const result = selector( ...args );
+		const resolver = resolvers[ reducerKey ] && resolvers[ reducerKey ][ key ];
```

Comment:
Minor: Good use case for `_.get`†

```js
const resolver = get( resolvers, [ reducerKey, key ] );
```

_† in that it arguably helps readability_